src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,13 @@
+$day-cell-side: 100px;
+$gap-size: 1px;
+$padding-size: 30px;
+$calendar-width: 7 * $day-cell-side + 6 * $gap-size + 2 * $padding-size;
```

Comment:
Better to save amount of days in variable, now it looks like magic number

Reply:
not quite clear with values 6 and 2. 
should they be saved in variables?

Reply:
@ArturNagaycev AFAIK 6 - is amount of days - 1, 2 can be an integer